packages/chat/components/chartElement/__stories__/examples.json

```diff
@@ -0,0 +1,383 @@
+
```

Comment:
I see we have `examples.json` and `examples2.json`, the latter being pulled into the `chartElement.stories.ts` file. Is `examples.json` being used anywhere?